### ISSUE TITLE
Adjusting index datetime to ticker timezone

### ISSUE THREAD

**Issue by rodrigobercini** [dpguthrie/yahooquery#17]
First of all, thank you so much for releasing this API, it's a great tool for people who study or work in the financial market.

**Is your feature request related to a problem? Please describe.**
When requesting stock price data, the dataframe returned will have the time according to GMT instead of the ticker's local timezone.
![image](https://user-images.githubusercontent.com/18664907/83906317-b9c6a800-a739-11ea-9324-a0ee01b51d85.png)

**Describe the solution you'd like**
Get ticker's local timezone as index when fetching stock price data.
Perhaps implementing this via an optional parameter.

```
aapl.history(interval='15m', start='2020-06-03', end='2020-06-06', adj_timezone=True)
aapl.history(interval='15m', start='2020-06-03', end='2020-06-06', adj_timezone=False)
```
**Describe alternatives you've considered**
It could be done by the user with feature engineering after querying.

**Additional context**
I think this could be a bigger issue for time zones that have big differences of time with GMT, since not only the time would be affected, but also the date.

**Comment by dpguthrie** [comment]
Thanks for noticing this.  I think this could be a relatively easy fix to adjust the index to the ticker's timezone.  Part of the data received through this API is a GMT offset.  I think it could be solved with the following:

```
df.index = pd.to_datetime(data[symbol]['timestamp'], unit='s') + \
               pd.Timedelta(data[symbol]['meta']['gmtoffset'], 's')
```

I can include it in the next release if you think that would solve the issue.

**Comment by rodrigobercini** [comment]
Awesome, I think that solves the issue.

Would you consider implementing this as an optional parameter? Having the index as GMT can be useful for people trading in different timezones and/or doing inter-market analysis.

**Comment by dpguthrie** [comment]
Yeah, I'll make it as an optional parameter that will default to setting the time to the ticker's timezone.

**Comment by dpguthrie** [comment]
This feature is available in 2.2.5 as an additional argument to the `history` method.

```
Ticker('aapl').history(adj_timezone=False)
```

If nothing is passed, it will default to True and the ticker's time will be used.

**Comment by rodrigobercini** [comment]
Awesome! I just ran some tests and this new parameter solves this issue.

Thank you!
